UI/window-basic-main.cpp

```diff
@@ -7862,6 +7862,16 @@ OBSProjector *OBSBasic::OpenProjector(obs_source_t *source, int monitor,
 	if (monitor > 9 || monitor > QGuiApplication::screens().size() - 1)
 		return nullptr;
 
+	bool closeProjectors = config_get_bool(GetGlobalConfig(), "BasicWindow",
+					       "CloseExistingProjectors");
+
+	if (closeProjectors && monitor > -1) {
+		for (size_t i = 0; i < projectors.size(); i++) {
+			if (projectors[i]->GetMonitor() == monitor)
+				DeleteProjector(projectors[i]);
```

Comment:
This code cannot close the _i_+1st display if the _i_th display is closed.
I'd like to recommend searching from `projector.size()-1` or add `i--` after deleting the _i_th projector.